DirectEmployers/MyJobs
# [PD 2960] gui for report intention

Task was to build a UI for reporting intentions so that the user has a good starting point of where they want to go.

To Test:

Pull down the repository and go to reports in the topbar. You should see the starting point includes PRM Reports, Job Analytics, and Site Analytics. Clicking on them will take you to the correct page. The switching to dynamic reporting facet on the right should be gone as well. However, site analytics currently doesn't go anywhere and Jason Sole is aware of that so clicking on it currently won't do anything. PRM and Job Analytics work however.
